src/app/components/chat/chat-detail/chat-detail.component.ts

```diff
@@ -111,7 +92,19 @@ export class ChatDetailComponent implements OnInit {
     });
   };
 
-  sendMessage = () => {
-    console.log(this.msgInput);
+  onSendMessage = () => {
+    const { uid } = this.authSvc.authInfo$.value,
+      { chatGroupId, msgInput, fbSvc, afs } = this;
+
+    const messageData: MessageI = {
+      id: afs.createId(),
+      chatGroupId,
+      senderId: uid,
+      content: msgInput,
+      createdAt: firestore.Timestamp.fromDate(new Date()),
+      // revert back to this when using the database
+      // createdAt: fbSvc.fsTimestamp(),
+    };
+    this.chatSvc.sendMessage(messageData);
```

Comment:
For a better UX we should clear out the contents of the input box after the message is sent.
```suggestion
    this.chatSvc.sendMessage(messageData);
    this.msgInput = '';
```